Specs/Scene/GroundPrimitiveSpec.js

```diff
@@ -565,7 +565,11 @@ defineSuite([
             classificationType : ClassificationType.TERRAIN
         });
 
-        verifyLargerScene(smallRectanglePrimitive, [255, 255, 255, 255], smallRectangle);
+        var largeScene = createScene({
+            canvas : createCanvas(2, 2)
+        });
+        verifyLargerScene(largeScene, smallRectanglePrimitive, [255, 255, 255, 255], smallRectangle);
+        largeScene.destroyForSpecs();
```

Comment:
Would it be safer to put all `largeScene` tests in their own sub-describe block with it's own beforeEach/afterEach so that cleanup always happens even if the test fails?

Reply:
Sure